### PR TITLE
Create the condition for non-list_view_conditions to work

### PR DESCRIPTION
**Pull request by rememberlenny**
# What

Add helper to check for details view beta. Keep assets in the details page if list view is deactivated.

### CODE REVIEW

File: app/views/proposals/show_next.html.haml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+- if !list_view_conditions && detail_view_conditions
```
Comment (phirefly) [inline app/views/proposals/show_next.html.haml]: Wouldn't this break in our dev environments then?

Comment (phirefly) [inline app/views/proposals/show_next.html.haml]: ^^ meaning if we have list view and detail view turned on in our dev/staging

Comment (rememberlenny) [inline app/views/proposals/show_next.html.haml]: @phirefly Then this kicks in: https://github.com/18F/C2/blob/master/app/views/layouts/application.html.haml#L17-L18

**Comment by phirefly** [comment]
:shipit:

**Comment by phirefly** [comment]
Verified through manual QA

**Pull request merged**